### PR TITLE
Add the script for debugging inconsistent bundle lists.

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose

- https://broadinstitute.atlassian.net/jira/software/projects/GH/boards/530?selectedIssue=GH-563

### Changes

- Add the helper script.
- Note: I intentionally re-used some code from the `run_analysis` package to avoid spending too much efforts on refactoring this time, but we shall re-visit later!

### Review Instructions

- Try it with some command like: `python get_bundle_diff.py -d prod -p 4a95101c-9ffc-4f30-a809-f04518a23803 -c caas_prod_key_for_prod.json`

### CODE REVIEW

**Comment by samanehsan** [comment]
I think this script could live in the `run_analysis` directory since it helps us identify which bundles we need to re-send to Lira for analysis. And then, you could import any functions that are already in `utils.py` like `format_bundle` and `get_bundles`

File: operations/debug_notifications/get_bundle_diff.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    print(f"Found {len(uuids_in_dss)} bundles in DSS.")
+
+    print(
+        f'Found {len(set(uuids_in_dss) - set(uuids_triggered_workflows))} bundles that in DSS but did not trigger workflows'
+    )
+
+    results = {
+        'bundles_in_dss': uuids_in_dss,
+        'bundles_triggered_workflows': uuids_triggered_workflows,
+        'diff': list(set(uuids_in_dss) - set(uuids_triggered_workflows)),
```
Comment (samanehsan) [inline operations/debug_notifications/get_bundle_diff.py]: I'd suggest saving `set(uuids_in_dss) - set(uuids_triggered_workflows)` to a variable (e.g. `bundles_diff`) and then reuse that above.

Comment (rexwangcc) [reply operations/debug_notifications/get_bundle_diff.py]: Done!

**Comment by rexwangcc** [comment]
Based on an offline discussion, moving the script under `run_analysis` will require updating the Cromwell-tools version the other script is using, and should not block this script from merging in. I will do that in a follow-up PR.

**Pull request merged**